submissions.html

```diff
@@ -0,0 +1,149 @@
+
+  We welcome papers analysing/reporting on scientific, technical, artistic and/or practice-based developments in <strong>Quantum Computer Music</strong>. Topics may include, but are not limited to:
+
+  <ul>
+    <li>Quantum Algorithms in Music</li>
+    <li>Quantum-computing Aided Composition</li>
+    <li>Quantum Representations of Audio</li>
+    <li>Quantum Signal Processing</li>
+    <li>Musical Instruments</li>
+    <li>Quantum Computing Software for Music</li>
+    <li>Sonification of Quantum Processes and Phenomena</li>
+    <li>Live performance with Quantum Computing</li>
+    <li>Music Information Retrieval</li>
+    <li>Hybrid Quantum-Classical Architectures</li>
+    <li>Music Theory and Quantum Computing Intersections</li>
+    <li>Quantum Aesthetics</li>
+    <li>Brain Quantum Computer Interfaces</li>
+    <li>Quantum Machine Learning</li>
+    <li>Quantum Natural Language Processing</li>
+    <li>Music Industry Applications</li>
+    <li>Quantum Computational Musicology</li>
+    <li>Quantum Computer Music in Education</li>
+  </ul>
+      </li>
+ 
+      <br>
+      All paper submissions will be peer-reviewed for originality, academic standards, appropriateness of topic, readability (language), and organisation.<br><br>
+
+      <li>Musical Works: We also welcome short (fixed) musical compositions and/or performances and/or demonstrations that showcase the intersection of quantum computing and musical creativity. For this route, a detailed <strong>manuscript</strong> explaining the idea must be submitted. I the work is accepted, it will be diffused/performed during the conference, at our conference auditorium - <a href='https://www.unipa.it/servizi/cerimoniale/concessione-spazi-steri/Spazi_Steri/'><u>Salla Delle Capriate</u></a><br><br>
+      The manuscript should not exceed 5 pages, and it should include:
+      <ul>
+        <li>Title of the work, author(s) name(s), and affiliation(s) (if applicable).</li>
+        <li>Abstract (max 200 words).</li>
+        <li>Artistic concept and description of the work.</li>
+        <li>Relevant technical details of the work.</li>
+        <li><em><strong>A Clear description of the conceptual and/or technical relationship between the work and quantum computation.</strong></em>Please note that quantum-inspired works are <strong>not direct synonyms</strong> of quantum computer music nor quantum computation-inspired work. We expect works that include quantum computation either technically, conceptually, or aesthetically.</li>
+        <li>In case of a live performance, include a tech rider, specifying your minimum requirements and the equipment you will bring.</li>
+        <li>Links to audio/video examples of the work (if available).</li>
+        <li>Include references to any relevant supporting literature whenever possible.</li>
+     </ul>
```

Comment:
it feels these are mostly specific submission instructions that should be below.
some things in the middle though could stay here, namely those that explain what is a music submission.